### PR TITLE
Include protocol in Capybara::Poltergeist::Inspect#url

### LINKED ISSUE #579: Inspector won't work with open command
Running
```open //localhost:9494```
returns
```The file //localhost:9494 does not exist.```

i think Capybara::Poltergeist::Inspect#url should contains "http:" prefix for url method.

### PR DESCRIPTION
**Pull request by bbuchalter**
Fixes #579

### CODE REVIEW

**Comment by bbuchalter** [comment]
Sadly it looks like the build is failing generally. Please feel free to ping this PR if my build fails after master is passing and I'll rework.

File: lib/capybara/poltergeist/inspector.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@ def browser
     end
 
     def url
-      "//localhost:#{port}/"
+      "http://localhost:#{port}/"
```
Comment (simi) [inline lib/capybara/poltergeist/inspector.rb]: It can be `https` also.

**Comment by simi** [comment]
Also this is revert of https://github.com/teampoltergeist/poltergeist/commit/9b73d618688c5cbdeaea600a193a514c58a754e6.

**Comment by nateberkopec** [comment]
#579 notes that this stops #open from working in general for most people. I understand this may cause insecure content warnings for some, but I think #open not working at all is worse. In favor of merging for that reason.

**Comment by edgarjs** [comment]
I agree that `#open` not working at all is worse. Insecure notices may be handled by setting `OpenSSL::SSL::VERIFY_PEER = OpenSSL::SSL::VERIFY_NONE` in your test environment (this is ruby specific but it could be done in any language I think).

**Comment by route** [comment]
Thanks for the PR, it should be fixed in master now.

**Pull request closed without merging**